### ISSUE TITLE
Build fails: cuda 11.4.2-devel-ubuntu20.04 not found

### ISSUE THREAD

**Issue by robdaly** [openhackathons-org/nways_accelerated_programming#14]
Cannot build [nways_Docker_python](https://github.com/openhackathons-org/nways_accelerated_programming/blob/9381b0d54548318116303340949e40f82dd17f15/nways_Dockerfile_python) as nvidia/cuda:11.4.2-devel-ubuntu20.04` is no longer available from docker.io:

```
docker pull nvidia/cuda:11.4.2-devel-ubuntu20.04
Error response from daemon: manifest for nvidia/cuda:11.4.2-devel-ubuntu20.04 not found: manifest unknown: manifest unknown
```

**Comment by aswkumar99** [comment]
Solved with #15

**Comment by robdaly** [comment]
Thanks, @aswkumar99 !
